Generator/Assets/Flags.cs

```diff
@@ -382,6 +382,12 @@ public class Flags
             { 0x8, 0x49 }, // Snowpeak summit cs.
             { 0x8, 0x45 }, // Snowpeak Summit intro CS.
         };
+        public static readonly byte[,] OpenDMTRegionFlags = new byte[,]
+        {
+            { 0x3, 0x81 }, // moved death mountain rock to exit
```

Comment:
I think this should be { 0x3, 0x79 }.
To the game, the region flag is defined as 0x26 which is `0b001 00110`
which is word 1, bit 6 relative to offset 0x8 in the region data.

This is the byte at offset 0xF ANDed with 0x40.

Encoding this with the system that is used in this file, you would be shifting 0x80 to the right by 1, and the byte offset is 0xF.
So that would be `0b01111 001` or 121 decimal or 0x79 hex.